lib/peep.rb

```diff
@@ -0,0 +1,25 @@
+require 'pg'
+
+class Peep
+  def self.all
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+    else
+      connection = PG.connect(dbname: 'chitter')
+    end
+
+    result = connection.exec("SELECT * FROM peeps;")
```

Comment:
A minor improvement here may be to use #exec_params to secure the app against SQL injection (like we saw in Challenge 12 of last week's pair programming challenge)